insecure/testHelpers.go

```diff
@@ -0,0 +1,89 @@
+package insecure
```

Comment:
This file does not contain any new changes, just encapsulating some of our test fixtures into a separate file.